src/io/varint.cpp

```diff
@@ -6,5 +6,5 @@ namespace fc
 void to_variant( const signed_int& var, variant& vo, uint32_t max_depth ) { vo = var.value; }
 void from_variant( const variant& var, signed_int& vo, uint32_t max_depth ) { vo.value = static_cast<int32_t>(var.as_int64()); }
```

Comment:
All looks good so far. But this opens the can of worms... signed_int is 32 bit and unsigned_int is 64. Perhaps a separate ticket.